### ISSUE TITLE
Some gotchas

### ISSUE THREAD

**Issue by herbdool** [backdrop-contrib/coder_upgrade#26]
There are a few gotchas which users should be aware of. Some of them can be fixed, others should have warnings. Currently using coder_upgrade could be misleading developers.

Bugs:

* Deletes `__DIR__`
* Needs `$config->save()` in upgrade hook
* Replaces `Drupal` even if in a URL
* Forces TRUE/FALSE to be strings in config file.

Misleading:

* If a config variable should instead be in `state_get`, users need to be aware. Perhaps a warning in the README file.

**Comment by bugfolder** [comment]
[PR 32](https://github.com/backdrop-contrib/coder_upgrade/pull/32) fixes the missing `$config->save()` in upgrade hook.

**Comment by bugfolder** [comment]
[PR 42 ](https://github.com/backdrop-contrib/coder_upgrade/pull/42) fixes "Forces TRUE/FALSE to be strings in config file.".

**Comment by roxy-jaqkar** [comment]
Trying to convert Bundle Copy module with Coder Upgrade and running into problems. The module is not recognizing my folders (new/old/patch). Have checked permissions, cleared cache and run cron to no avail. Any ideas?
![coder error](https://user-images.githubusercontent.com/44479907/140363065-187ea458-6007-4854-965f-b5ccef21e400.png)

**Comment by docwilmot** [comment]
Strange. To be clear, you placed the module folder to be converted into `SITE_ROOT/files/coder_upgrade/old` and not `SITE_ROOT/modules/coder_upgrade/old` ?

**Comment by roxy-jaqkar** [comment]
Yes, path is: site_root/files/coder_upgrade/old. I uploaded a zip file.

**Comment by docwilmot** [comment]
Ahh no, not a zipped file, upload the actual folder.

**Comment by roxy-jaqkar** [comment]
OK, will try that. Thanks!

**Comment by bugfolder** [comment]
Bringing this back on-topic, the two remaining open bugs from this issue are (adding checkboxes to check them off as they get addressed):

* [ ] ... | Deletes `__DIR__`
* [ ] ... | Replaces `Drupal` even if in a URL
